addons/website_event/models/event_tag.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class EventTagCategory(models.Model):
+    _name = 'event.tag'
+    _inherit = ['event.tag', 'website.published.mixin']
```

Comment:
Do we need the publish mixin on tags themselves ? Isn't it controlled by the category ? I see in specs that "​Add a website_id on the category so that only the right tags are displayed", so I suppose the category is the publish management record ?